soccer/radio/USBRadio.cpp

```diff
@@ -183,57 +185,15 @@ void USBRadio::send(Packet::RadioTx& packet) {
     uint8_t forward_packet[rtp::ForwardSize];
 
     // ensure Forward_Size is correct
-    static_assert(sizeof(rtp::Header) + 6 * sizeof(rtp::RobotTxMessage) ==
-                      rtp::ForwardSize,
-                  "Forward packet contents exceeds buffer size");
-
-    // Unit conversions
-    static const float Seconds_Per_Cycle = 0.005f;
-    static const float Meters_Per_Tick = 0.026f * 2 * M_PI / 6480.0f;
-    static const float Radians_Per_Tick = 0.026f * M_PI / (0.0812f * 3240.0f);
-
-    rtp::Header* header = (rtp::Header*)forward_packet;
-    header->port = rtp::PortType::CONTROL;
-    header->address = rtp::BROADCAST_ADDRESS;
-    header->type = rtp::MessageType::CONTROL;
-
-    // Build a forward packet
-    for (int slot = 0; slot < 6; ++slot) {
-        // Calculate the offset into the @forward_packet for this robot's
-        // control message and cast it to a ControlMessage pointer for easy
-        // access
-        size_t offset =
-            sizeof(rtp::Header) + slot * sizeof(rtp::RobotTxMessage);
-        rtp::RobotTxMessage* msg =
-            (rtp::RobotTxMessage*)(forward_packet + offset);
-
-        if (slot < packet.robots_size()) {
-            const Packet::Control& robot = packet.robots(slot).control();
-
-            msg->uid = packet.robots(slot).uid();
-            msg->messageType = rtp::RobotTxMessage::ControlMessageType;
-
-            auto& controlMessage = msg->message.controlMessage;
-
-            controlMessage.bodyX = static_cast<int16_t>(
-                robot.xvelocity() * rtp::ControlMessage::VELOCITY_SCALE_FACTOR);
-            controlMessage.bodyY = static_cast<int16_t>(
-                robot.yvelocity() * rtp::ControlMessage::VELOCITY_SCALE_FACTOR);
-            controlMessage.bodyW = static_cast<int16_t>(
-                robot.avelocity() * rtp::ControlMessage::VELOCITY_SCALE_FACTOR);
-
-            controlMessage.dribbler =
-                clamp(static_cast<uint16_t>(robot.dvelocity()) * 2, 0, 255);
-
-            controlMessage.kickStrength = robot.kcstrength();
-            controlMessage.shootMode = robot.shootmode();
-            controlMessage.triggerMode = robot.triggermode();
-            controlMessage.song = robot.song();
-        } else {
-            // empty slot
-            msg->uid = rtp::INVALID_ROBOT_UID;
-        }
-    }
+    //static_assert(sizeof(rtp::Header) + 6 * sizeof(rtp::RobotTxMessage) ==
+    //                  rtp::ForwardSize,
+    //              "Forward packet contents exceeds buffer size");
```

Comment:
I would almost say we just kill this file since we aren't using those radios anymore. It'll make it easier to fix the packets

Reply:
I would be okay with that.